lnp2p/src/bolt/bolt2.rs

```diff
@@ -735,9 +734,46 @@ impl DumbDefault for AcceptChannel {
     }
 }
 
+impl lightning_encoding::LightningEncode for CommitmentSigned {
+    fn lightning_encode<E: io::Write>(&self, mut e: E) -> Result<usize, Error> {
+        self.channel_id.lightning_encode(&mut e)?;
+        self.signature.lightning_encode(&mut e)?;
+        let num_htlcs = self.htlc_signatures.len() as u16;
+        num_htlcs.lightning_encode(&mut e)?;
+        for index in 0..num_htlcs {
+            let sig: Signature = self.htlc_signatures[index as usize];
+            sig.lightning_encode(&mut e)?;
+        }
+
+        let buf: Vec<u8> = vec![];
+        e.write_all(&buf)?;
+        Ok(buf.len())
```

Comment:
Can you pls explain what this code tries to achieve?

Reply:
Return the number of bytes.

Reply:
It will always return zero. It does nothing